webapp/security/api.py

```diff
@@ -180,3 +180,22 @@ def get_sitemap_cves(self, limit, offset):
             raise SecurityAPIError(error)
 
         return sitemap_response.json()
+
+    def get_sitemap_notices(self, limit, offset):
+
+        parameters = {"limit": int(limit), "offset": int(offset)}
+
+        parameters = {k: v for k, v in parameters.items() if v is not None}
```

Comment:
```suggestion
parameters = {}
if limit is not None:
    parameters["limit"] = int(limit)
if offset is not None:
    parameters["offset"] = int(offset)
```

Since int(None) will raise an error.

Reply:
Nice catch, thanks! About the limit, it actually doesn't need to be below 100 anymore. Being able to increase the limit without facing performance issues was one of the reasons behind having a dedicated endpoint. It only returns 2 fields and neither are heavy so it handles a high limit without issue